.github/workflows/stubtest_stdlib.yml

```diff
@@ -44,6 +44,6 @@ jobs:
           cache: pip
           cache-dependency-path: requirements-tests.txt
       - name: Install dependencies
-        run: pip install $(grep mypy== requirements-tests.txt)
+        run: pip install -r requirements-tests.txt
```

Comment:
Arguably only installing the 1-2 dependency we need for a single test doesn't sound like a bad idea either.

Reply:
Ah, it would be good to keep stubtest stdlib ones as they are. Unlike stubtest third-party, it doesn't create venvs on the fly, and it's useful to be 100% sure that it's running in as isolated an environment as possible

Reply:
But keep `-r requirements-tests.txt` fir third party right?

Reply:
Yup, the third-party one depends on `utils.py`, and we can be sure it's testing packages in isolation because of the way it creates venvs on the fly 👍